_api-reference/tasks.md

```diff
@@ -267,7 +267,7 @@ To associate requests with tasks for better tracking, you can provide a `X-Opaqu
 Usage:
 
 ```bash
-curl -i -H "X-Opaque-Id: 111111" "https://localhost:9200/_tasks" -u 'admin:admin' --insecure
+curl -i -H "X-Opaque-Id: 111111" "https://localhost:9200/_tasks" -u 'admin:< Admin password >' --insecure
```

Comment:
You will want to ask @kolchfa-aws etc. about the style guidelines for these placeholders. I am not sure whether leading/trailing spaces should be trimmed or whether underscores should be used instead of spaces in between the terms.

Reply:
Sorry for the late reply, just saw this. Generally we don't use spaces within angular brackets and separate the words with a hyphen or underscore so `<custom-admin-password>` looks good to me.